README.md

```diff
@@ -39,20 +39,73 @@ will contain QEMU for testing a virtualized platform.
 
 Intended for local use to develop for EDKII based UEFI products.
 
+## Using containers locally
+
+Containers can provide a convenient and consistent dev environment when building
+EDK2 based firmware projects. This section details some tools and tips that make
+using containers for local development easier. The DEV editions of the containers
+are intended for this purpose. This section is not comprehensive however and it
+is encouraged users experiment and consider contributing back any new useful
+configurations or tools to this documentation.
+
+__NOTE__: If your code base is cloned in Windows, it is not advised that you directly
+open this repository in a Linux dev container as the file system share between Windows
+and WSL 2 causes a very significant performance reduction. Instead, clone the
+repo in the WSL file system and map into the container or directly clone into the
+container.
+
+### Visual Studio Code
+
+The Visual Studio Code [Dev Container extension](https://code.visualstudio.com/docs/devcontainers/containers)
+provides an easy and consistent way to use containers for local development. At
+the time of writing, this extension only supports Linux based containers. This
+extension provides a number of useful additions to the specified docker image on
+creation.
+
+- Configures git credential manager to pipe in git credentials.
+- Makes extensions available on code inside the container.
+- Abstracts management of the container for seamless use in the editor.
+
+For a shared docker image configuration, this can be configured by creating a
+.devcontainer file in the repository. Some useful configurations are details below.
+
+| Configuration          | Purpose |
+| :------------          | :------ |
+| "privileged": true     | This may be needed for access to KVM for QEMU acceleration.  |
+| "forwardPorts": [####] | Can be used to forward debug or serial ports to the host OS. |
+
+It may also be desireable to run initialization commands using the "postCreateCommand"
+option. Specifically running "git config --global --add safe.directory ${containerWorkspaceFolder}"
+may be required if mapping the repository into the container is expected.
+
+And example of a devcontainer used for a QEMU platform repo is included below.
+
+```json
+{
+  "image": "ghcr.io/tianocore/containers/fedora-35-dev:latest",
+  "postCreateCommand": "git config --global --add safe.directory ${containerWorkspaceFolder} && pip install --upgrade -r pip-requirements.txt",
+  "forwardPorts": [5000],
+  "privileged": true
+}
+```
+
 ## Notes
 
 ### Ubuntu 20
+
```

Comment:
Why is the Ubuntu section still in this file? I thought I moved that to the sub-readme?
(not relevant for this PR)